cluster/kubernetes/resource/resource.go

```diff
@@ -198,7 +205,11 @@ func unmarshalKind(base baseObject, bytes []byte) (KubeManifest, error) {
 		return &list, nil
 	case base.Kind == "FluxHelmRelease" || base.Kind == "HelmRelease":
 		var fhr = FluxHelmRelease{baseObject: base}
-		if err := yaml.Unmarshal(bytes, &fhr); err != nil {
+		// NB: workaround for go-yaml/yaml/issues/139
+		// By using github.com/ghodss/yaml to unmarshal HelmReleases.
+		// This effectively results in all keys of `Value`s being strings
+		// and not interface{}.
+		if err := jsonyaml.Unmarshal(bytes, &fhr); err != nil {
```

Comment:
Does this mean the `mapper` stuff above can be removed?

Reply:
Above where? Quite a lot of 'mapper' things on this page...